scripts/get_raw_sun_data.py

```diff
@@ -0,0 +1,81 @@
+############
+# Pull raw sun data using pvlib
+#
+# 2021-09-01
+# Peter Dudfield
+#
+# The data is about
+# - 1MB for a 2 of days, for ~2000 sites and takes about ~1 minutes
+# - 6MB for a 10 of days, for ~2000 sites and takes about ~1 minutes
+# - 252MB for a 365 of days, for ~2000 sites and takes about ~11 minutes (on a macbook pro)
+
+# Decide to just go for one year of data
+# on 1st Jan 2019 and 2020, the biggest differences was in elevation was 1 degree,
+# More investigation has been done (link), and happy difference is less than 1 degree,
+# Therefore, its ok good to use 1 year of data, for all the years
```

Comment:
It might be worth commenting that, for now, this only works for GSPs (not individual PV systems).  (which is absolutely the right thing to do, given that we're focused on GSPs for now!)

Reply:
I think it still works for a PV system too, 
for context, the sun data is both GSP and PV locations

Reply:
cool beans, sorry, my mistake!